content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -785,6 +785,65 @@ spec:
       storage: 10Gi
 ```
 
+## Volume Populators and Data Sources
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+{{< note >}}
+Volume populars are available as alpha since Kubernetes 1.18, are redesigned in Kubernetes 1.22. The `AnyVolumeDataSource` feature must be enabled. Refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
+{{< /note >}}
+
+Volume populators take advantage of a new PVC spec field called `dataSourceRef`. Unlike the
+`dataSource` field, which can only contain either a reference to another PersistentVolumeClaim
+or to a VolumeSnapshot, the `dataSourceRef` field can contain a reference to any object in the
+same namespace, except for core objects other than PVCs.
+
+### Data Source Field Semantics
+
+Use of the `dataSourceRef` field is nearly identical to the `dataSource` field, in that filling
+in either one causes the other to be filled in with the same value, and neither can be changed
+after creation. The two fields will thus always have the same contents. The two differences that
+matter are:
```

Comment:
@sftim need an english native speaker to reword this, i feel like i need to think about this paragraph very hard to understand what it means but i'm also unable to come up with something better.